Cargo.toml

```diff
@@ -30,8 +30,8 @@ default = []
 backtraces = []
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "5cfcf1470b6df277a854d844f77979ab3b93f666", features = ["iterator","staking","stargate"] }
-cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", rev = "5cfcf1470b6df277a854d844f77979ab3b93f666", features = ["iterator","staking","stargate"] }
+cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "12ed32fc69d07a4b55b4c4d99c10e91a8434da90", features = ["iterator","staking","stargate"] }
```

Comment:
it will be changed after https://github.com/line/cosmwasm/pull/182 squash merged.